source/OptimaJet.DWKit.StarterApplication/Migrations/20181214051403_RenameSilInfrastructureToDefaultBuildEngine.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace OptimaJet.DWKit.StarterApplication.Migrations
+{
+    public partial class RenameSilInfrastructureToDefaultBuildEngine : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
```

Comment:
This migration doesn't appear to do anything?

Reply:
Yeah, I wasn't sure what to do about that.  When I ran `./run db:migrations add ...` it got created AND the tool make the changes to the previous migrations.  That seemed odd, but it was helpful?  It would not have been good if we had already put this into production.

So I could remove the new files.

Reply:
Removing the files